app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -156,10 +156,18 @@ class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
       if (it) {
         dialogShower.show(
           WifiOnly,
-          { sharedPreferenceUtil.putPrefWifiOnly(false) },
-          { onRefreshStateChange(false) }
+          {
+            sharedPreferenceUtil.putPrefWifiOnly(false)
+            zimManageViewModel.shouldShowWifiOnlyDialog.value = false
+          },
+          {
+            onRefreshStateChange(false)
+            context.toast(
+              resources.getString(R.string.denied_internet_permission_message),
+              Toast.LENGTH_SHORT
```

Comment:
@MohitMaliFtechiz Do we need to show the message again?

Reply:
Its better to notify to enable wifi if they select No .